src/app/commons/token.interceptor.ts

```diff
@@ -23,6 +26,15 @@ export class TokenInterceptor implements HttpInterceptor {
         }
       });
     }
-    return next.handle(request);
+
+    return next.handle(request).pipe(tap(() => { },
```

Comment:
() => {} at first tap arg, what is this doing?

Reply:
first part is related to success
second part (after " , ") is related to errror
since this handler's job is to handle error, there is no reason to do anything on success